samples/bluetooth/channel_sounding_ras/reflector/prj.conf

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+CONFIG_DK_LIBRARY=y
+
+CONFIG_BT=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_SMP=y
+CONFIG_BT_DEVICE_NAME="Nordic CS Reflector"
+CONFIG_BT_MAX_CONN=1
+
+# The Ranging Profile recommends a MTU of at least 247 octets.
+CONFIG_BT_L2CAP_TX_MTU=498
+CONFIG_BT_BUF_ACL_TX_SIZE=502
+CONFIG_BT_BUF_ACL_RX_SIZE=502
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
```

Comment:
is DLE needed ?

Reply:
It's selected by defining a larger TX and RX size:
https://github.com/nrfconnect/sdk-zephyr/blob/e4c9dffbbd4e5c25504da8d76159c9b411982e96/subsys/bluetooth/Kconfig#L150